test_runner/regress/test_disk_usage_eviction.py

```diff
@@ -264,8 +266,8 @@ def test_pageserver_falls_back_to_global_lru(eviction_env: EvictionEnv):
     assert actual_change >= target, "eviction must always evict more than target"
 
     time.sleep(1)  # give log time to flush
-    assert env.neon_env.pageserver.log_contains("falling back to global LRU")
-    env.neon_env.pageserver.allowed_errors.append(".*falling back to global LRU")
+    assert env.neon_env.pageserver.log_contains(GLOBAL_LRU_LOG_LINE)
+    env.neon_env.pageserver.allowed_errors.append(".*" + GLOBAL_LRU_LOG_LINE)
 
 
 def test_partial_evict_tenant(eviction_env: EvictionEnv):
```

Comment:
This failed on second to last assert: https://neon-github-public-dev.s3.amazonaws.com/reports/pr-3890/release/4545869605/index.html#suites/0e58fb04d9998963e98e45fe1880af7d/f442ab995741e498/

```
assert 31555584 > (0.4 * 82632704)
```

Where rhs is: `33053081.6` and left side is 38% of the total.

I wonder if this test should have checkpointing after pg destroy.

But unsure what could be the cause here. We probably do not log the debug for these tests?

Reply:
Is the select only workload (`-S`) from pgbench expected to do something? Looking at the logs there is no new layers which would match the read-only nature.

Reply:
Done: https://github.com/neondatabase/neon/pull/3890/commits/b47a02569f04c36c938d17cb4fd9de4267c10254